### ISSUE TITLE
Loritta mentioning role that don't have permission to use commands

### ISSUE THREAD

**Issue by mitsuaky** [LorittaBot/Loritta#2312]
![image](https://user-images.githubusercontent.com/42049025/102566055-715f4e00-40bd-11eb-90cc-0cf0ea51fb40.png)
She has admin, so the problem is, she's actually mentioning 1k+ people everytime someone with this role mentions her.

**Comment by MrPowerGamerBR** [comment]
While @iiAmFilippi made a fix by changing the command to use the role name, I think the proper prettier fix is by using the "Allowed Mentions" feature.

```
val builder = MessageBuilder()
                            .setAllowedMentions(listOf(Message.MentionType.USER, Message.MentionType.CHANNEL))
                            .setContent(messageHere)
                            .build()
```

This would make the message show the role mention, but no one from the role would be mentioned.

**Comment by mitsuaky** [comment]
A similar problem occurs with the command +rep. If the user try to use +rep @everyone or @here, she will mention everyone if she has permission.

![image](https://user-images.githubusercontent.com/42049025/103175617-05fa4680-484a-11eb-8e55-3e5504a440dc.png)

**Comment by iiAmFilippi** [comment]
> 
> 
> A similar problem occurs with the command +rep. If the user try to use +rep @everyone or @here, she will mention everyone if she has permission.
> 
> ![image](https://user-images.githubusercontent.com/42049025/103175617-05fa4680-484a-11eb-8e55-3e5504a440dc.png)

This problem has already been fixed in [a25ce06](https://github.com/LorittaBot/Loritta/commit/a25ce06f023317c08ae31f7f1a2732a619ff6022).
